src/age_types.rs

```diff
@@ -108,7 +128,7 @@ where
             return Err("unsupported JSONB encoding version".into());
         }
 
-        // Remove ::vertex from bytes
+        // Remove ::edge from bytes
```

Comment:
Magic numbers. Maybe move it to the constant?

Reply:
I'm pretty sure this isn't a magic number?
Do you mean the existing line below?